omui/src/components/checkbox/checkbox.style.ts

```diff
@@ -1,34 +1,32 @@
-import { ComponentTheme, mode, Props } from '@chakra-ui/theme-tools';
+import theme from '@chakra-ui/theme';
+import { BaseStyle, mode } from '@chakra-ui/theme-tools';
 
-function checkedStyle(props: Props) {
-  const { colorScheme: c } = props;
-  return {
-    bg: mode(`${c}.500`, `${c}.200`)(props),
-    borderColor: mode(`${c}.500`, `${c}.200`)(props),
-    color: mode('white', 'gray.900')(props),
-    _hover: {
-      bg: mode(`${c}.600`, `${c}.300`)(props),
-      borderColor: mode(`${c}.600`, `${c}.300`)(props)
-    },
-    _disabled: {
-      borderColor: mode('gray.200', 'transparent')(props),
-      bg: mode('gray.200', 'whiteAlpha.300')(props),
-      color: mode('gray.500', 'whiteAlpha.500')(props)
-    }
-  };
-}
+const { Checkbox } = theme.components;
 
-const baseStyle = (props: Props) => {
+const baseStyle: BaseStyle<typeof Checkbox.register> = (props) => {
```

Comment:
I think I'm actually fine to use the checkbox styles in @chakra-ui/theme here instead.